scripts/build/builders/bouffalolab.py

```diff
@@ -115,29 +127,69 @@ def __init__(self,
         self.argsOpt.append('baudrate=\"{}\"'.format(baudrate))
 
         if bouffalo_chip == "bl602":
-            self.argsOpt.append('chip_enable_openthread=false')
-            self.argsOpt.append('chip_enable_wifi=true')
-        if bouffalo_chip == "bl702":
+
+            enable_wifi = True
```

Comment:
Add these options of `-wifi`, `-ethernet`, `-thread` are for BL706 SoC, and the developing BL616 Soc (which has Wi-Fi 6, Thread and Ethernet on chip). 
- To be compatible with BL706/BL616 options, appending `-wifi` for BL602 target will not break the building.
- Since BL602 only has Wi-Fi connectivity, it more feels duplicate to be must add `-wifi`. And our customers are used to  BL602 target without `-wifi`.